12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,90 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+  #amir@ubuntu:~/wildcards$ find ./ -name "[[:lower:]]*.*"
+  #./Doc/test.c
+  #./lib/zlib.c
+  #./img/hero.png
+  #./img/player.png
+  #./src/player.c
+  #./src/main.c
+  #./include/player.h
+
 
   2. All files whose name starts with an uppercase letter
   Answer:
 
+  # amir@ubuntu:~/wildcards$ find ./ -name "[[:upper:]]*.*"
+  # ./Doc/README.txt
+
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
+  # amir@ubuntu:~/wildcards$ find ./ -name "*[aeiou].*"
+  #./img/hero.png
 
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+  # amir@ubuntu:~/wildcards$ find ! -name  "*[aeiou].*"
+  #.
+  #./Doc
+  #./Doc/test.c
+  #./Doc/README.txt
+  #./lib
+  #./lib/zlib.c
+  #./img
+  #./img/player.png
+  #./src
+  #./src/player.c
+  #./src/main.c
+  #./include
+  #./include/player.h
+
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
+  #amir@ubuntu:~/wildcards$ find ./src  -name "*.[c]"
+  #./src/player.c
+  #./src/main.c
 
-  6. All files with extension ".c" under any folder
+
+  6. All files with extension ".c" under any folder 
   Answer:
+  #  amir@ubuntu:~/wildcards$ find ./  -name "*.[c]"
+  #./Doc/test.c
+  #./lib/zlib.c
+  #./src/player.c
+  #./src/main.c
+
 
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
+  #find ./  -name "*.png" -exec rm -f {} \;
```

Comment:
Correct! Though I'd use "rm -i" instead so you get a prompt before deleting each file and so can be sure you're not deleting things you need